src/components/value-list/value-list.tsx

```diff
@@ -192,6 +226,15 @@ export class ValueList<
     event.stopPropagation();
   }
 
+  @Listen("calciteInternalListItemDragHandleFocused")
+  handleDragHandleFocus(event: FocusEvent): void {
+    const item = getItemElement(event);
+    const handleElement = getHandleElement(event.detail as any);
```

Comment:
Maybe this event can just pass the element instead of the event? That would clean up the `any` as well.